tests/test_access_route.py

```diff
@@ -42,9 +43,9 @@ def test_malformed_rfc_forwarded(self):
             headers={
                 'Forwarded': 'for'
             }))
-        self.assertEqual(req.access_route, ['127.0.0.1'])
+        self.assertEqual(req.access_route, [])
```

Comment:
Don't fall back to REMOTE_ADDR when the field is malformed; expose the problem with an upstream proxy to the app (and the operator of said app).